pub fn exit_with_error(message: &str) {
    use std::process;

    println!("{}", message);
    println!("abortingâ€¦");

    process::exit(1);
}
